app/flows/check_uk_visa_flow/outcomes/outcome_no_visa_needed.erb

```diff
@@ -3,6 +3,10 @@
 <% end %>
 
 <% govspeak_for :body do %>
+  <%if calculator.passport_country_requires_electronic_travel_authorisation?  %>
+    <%= render partial: 'electronic_travel_authorisation_advice', locals: {calculator: calculator} %>
+  <% end %>
+
   <% if (calculator.travelling_to_elsewhere? || calculator.travelling_to_ireland?) &&
```

Comment:
I wonder if formatting this a little and giving it extra brackets would help it's readability... 🤔 

```ruby
<% if (
    (calculator.travelling_to_elsewhere? || calculator.travelling_to_ireland?) && 
    (calculator.passport_country_in_non_visa_national_list? || 
        calculator.passport_country_in_eea? || 
        calculator.passport_country_in_british_overseas_territories_list?)
) %>
```

Reply:
IDE complained about having parentheses around a conditional. But I am incorporating the breaking of lines as you have suggested 👍